package.json

```diff
@@ -3,7 +3,7 @@
 	"displayName": "Apache Daffodil VS Code Extension",
 	"description": "VS Code extension for Apache Daffodil DFDL schema debugging",
 	"version": "1.1.0-SNAPSHOT",
-	"daffodilVersion": "3.3.0",
+	"daffodilVersion": "3.4.0",
```

Comment:
We made some updates to he NOTICE/LICENSE files as part of Daffodil 3.4.0: https://github.com/apache/daffodil/commit/60e5f98abbeaa2fb8c94e5d64927a5c840ce2082

Some of those don't need to be carried over to he VS Code `build/package` files since they are CLI dependencies and the don't get pulled into VS Code (e.g. Exificient library), but I *think* some of them do need to be included.

Reply:
@stevedlawrence Would I just need to add in the updated NOTICE/LICENSE for the libraries included in the `-bin.zip` file from the 3.4.0 release that aren't currently listed?

Reply:
> Would I just need to add in the updated NOTICE/LICENSE for the libraries included in the -bin.zip file from the 3.4.0 release that aren't currently listed?

Yep, that's correct. The NOTICE/LICENSE changes that I pointed to in that commit are for the Daffodil CLI, which includes license information for a bunch of jars. But VS Code doesn't depend on he CLI, it depends on daffodil-japi, which transitively pulls in just a subset of what the CLI uses. So some of those NOTICE/LICENSE changes don't affect VS Code since it doesn't pull them in and redistribute them.

Reply:
@stevedlawrence So between the daffodil-debugger zip and the daffodil bin zip I found all these items

```txt
1.) Listed in daffodil-debugger zip, need added:
    - com.fasterxml.woodstox.woodstox-core
    - com.google.code.gson.gson
    - com.monovore.decline-effect
    - com.monovore.decline
    - com.typesafe.config
    - io.reactivex.rxjava2.rxjava
    - org.typelevel.cats-effect-kernel
    - org.typelevel.cats-effect-std
    - org.typelevel.cats-effect
    - org.typelevel.literally
    - org.typelevel.log4cats-core
    - org.typelevel.log4cats-slf4j

2.) Listed in daffodil 3.4.0, may need added:
    - org.apache.daffodil.daffodil-cli
    - org.apache.daffodil.daffodil-japi
    - org.apache.daffodil.daffodil-runtime2
    - org.apache.daffodil.daffodil-tdml-lib
    - org.apache.daffodil.daffodil-tdml-processor

3.) Listed in daffodil 3.4.0, may not need added (listed as part of daffodil-cli deps):
    - com.siemens.ct.exi.exificient
    - com.siemens.ct.exi.exificient-core
    - com.siemens.ct.exi.exificient-grammars
    - com.sun.xml.bind.jaxb-core
    - com.sun.xml.bind.jaxb-impl
    - commons-codec.commons-codec
    - javax.activation.activation
    - javax.xml.bind.jaxb-api
    - net.sf.saxon.Saxon-HE
    - org.apache.daffodil.daffodil-schematron
    - org.apache.httpcomponents.client5.httpclient5
    - org.apache.httpcomponents.core5.httpcore5
    - org.apache.httpcomponents.core5.httpcore5-h2
    - org.apache.logging.log4j.log4j-core
    - org.fusesource.jansi.jansi
    - org.jline.jline
    - org.rogach.scallop
    - org.xmlresolver.xmlresolver
    - xmlpull.xmlpull
```

Would I just need the dependencies listed in 1 or 1 and 2? I assume the deps listed in 3 do not need added as they seemed to be related to daffodil-cli correct? Also would you like them added as part of this PR or a separate PR?

Reply:
Correct, you only need the deps in 1 and 2. I would prefer license/notice information be added in this PR since it is a related change.

Reply:
@stevedlawrence @mbeckerle Can you give another look over his PR as I have added the necessary dependencies to our lists. All were ALv2 licensed, most not having a NOTICE file.